contents/spotlight/startup-before-sunset-ai.md

```diff
@@ -0,0 +1,71 @@
+
+[BeforeSunset AI](https://www.beforesunset.ai/) is an AI-powered daily planner that takes care of your schedule and helps ward off the stress of missed deadlines and forgotten tasks. Founded in November 2022 by entrepreneur and software engineer Elif Duran, BeforeSunset has since grown to a team of 15 and is used by hundreds of teams. 
+
+“The first version of BeforeSunset was just an internal time tracking tool I’d built,” explains Elif. “But when we launched as a separate venture we saw that no other productivity tool asks you the questions needed to plan a day properly. We realized that AI could help if it had the right information.”
+
+After rebuilding the product around this AI-powered vision, the team wanted to relaunch in earnest — and in June 2023 debuted on Product Hunt, going on to win [Product of the Day, Week, and Month](https://www.producthunt.com/products/beforesunset). Here’s how:
+
+![BeforeSunset AI Launch on Product Hunt](../images/blog/before-sunset-ai.png)
+
+### 1. Create a product launch plan in advance
+
+Unsurprisingly for a productivity startup, it all began with a plan.
+
+“We had a clear plan in our minds for the product,” says Elif. “We wanted to complete the core features, such as to-do listing and time blocking, and we also wanted to build in AI features. In total it took about three months before we were ready to launch to users.”
+
+This development time afforded the growth team time to plan, while also giving them the benefit of knowing how the product would come together. Work was hard — to the point that Elif worked several 40-hour sprints without sleeping — but came together smoothly. 
+
+“On the marketing side our efforts started six months before the launch,” says Growth Lead Zekiye Nur Kesici. “In addition to on-going activities we began a waitlist for the new version and began growing it. We tried many different channels, including Reddit, Indie Hackers, Hacker News — and we spent a month on competitor analysis to find new channels too.”
+
+> **Putting it into action:** For new product launches take steps to align your product and marketing team as early as possible, and make sure they stay in sync as development continues. Product managers should keep marketing teams informed of key features and delays, so marketing can plan how to position the launch and manage deliverables. Product marketers, unsurprisingly, can help bridge the gap. 
+
+### 2. Prioritize community and creativity
+
+Focusing on community engagement is an important part of any launch plan, according to Zekiye — and the earlier you start, the more successful you’ll be. 
+
+“We chose to focus on the Product Hunt community and we knew that would require us to be active on the platform,” she explains. “That means finding ways to add value; ways you can support them whenever you’re in touch so that, in turn, you can benefit from the community.”
```

Comment:
```suggestion
“We chose to focus on the Product Hunt community and we knew that would require us to be active on the platform,” she explains. “That means finding ways to add value; ways you can support them whenever you’re in touch so that, in turn, you can benefit from the community.”

For BeforeSunset, this included engaging with the community regularly and offering expertise where possible. 
```